docs/_milestone_.md

```diff
@@ -0,0 +1,85 @@
+# December Milestone
+
+## Overview
+
+- Complete the epics:
+  - `Add Support for Partitioned Database`
+  - `Inclusion of related models`
+  - `Production deployment`
+
+## Goals
+
+- [ ] Inclusion of related models [MVP]
+      https://github.com/strongloop/loopback-next/issues/1352
+
+  - [ ] [5]Reject create/update requests when data contains navigational
+        properties https://github.com/strongloop/loopback-next/issues/3439
+
```

Comment:
If all goes well, the MVP scope will be done in November 🤞We may want to consider the following post-MVP story because it already has 5 upvotes:

- [NEEDS ESTIMATE] Include related models with a custom scope https://github.com/strongloop/loopback-next/issues/3453

Reply:
Per our discussion, unfortunately we need to commit less for December due to the holidays and vacation time.  But this will be high on the list for Jan for sure. 
cc @mamillar93

Reply:
will move to stretch goals

Reply:
Thanks friends!